stories/locfeature.IMMER.mdx

```diff
@@ -24,11 +24,179 @@ taxonomy:
       - Space for Earth
 ---
 
-import Carousel from "../overrides/common/embedded-video-carousel";
-import contentArray from './locfeature.IMMER/carousel_content.json';
+import CardGallery from "./components/card_gallery";
+import { visitLocationFeaturesStoryIds } from '../overrides/common/story-data';
 
-<Block type='wide'>
+<Block >
+    <Prose>
+      ## Visit Space for Earth
```

Comment:
One nitpick: can you review the use of headings so they have a hierarchy? (https://www.w3.org/WAI/tutorials/page-structure/headings/) This page only has h2 and h4 now.

Reply:
just saw this, done!